optimum/exporters/onnx/base.py

```diff
@@ -803,7 +803,9 @@ class OnnxConfigWithLoss(OnnxConfig, ABC):
         "default": {"labels": {0: "batch_size"}},
         "masked-lm": {"labels": {0: "batch_size", 1: "sequence_length"}},
         "causal-lm": {"labels": {0: "batch_size", 1: "sequence_length"}},
+        "causal-lm-with-past": {"labels": {0: "batch_size"}},
```

Comment:
It could be kind of difficult to handle all those cases in a single `OnnxConfigWithLoss` no? Is there a reason for the asymmetry compared to `OnnxConfigWithLoss`?

I am not sure to understand why `OnnxConfigWithLoss` is needed in the first place. It appears to just add labels / loss in the inputs/outputs.

Reply:
As I explained.